docs/products/metrics/howto/download-influxdb-data.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Download Aiven for InfluxDB data backup
+sidebar_label: Download InfluxDB data
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+Download a backup of your Aiven for InfluxDB service data by requesting a backup. This creates a snapshot of your latest data, ready for download.
+
+## Prerequisites
+
+Before you download a backup, make sure you have the following:
+
+- [Aiven API token](/docs/platform/howto/create_authentication_token) with the necessary
+  permissions
+- [Aiven CLI](/docs/tools/cli)
+
+If your InfluxDB service is in a Virtual Private Cloud (VPC) or if you need specific
+network access to your backup, confirm that the correct access settings are enabled.
+Based on your network setup, enable one of the following settings:
+
+- `public_access.user_backup`: Allows access to backups from the public internet
+- `private_access.user_backup`: Allows access to backups within a private network
+- `privatelink_access.user_backup`: Allows access to backups via a private link
+
+To configure access, go to the service's <ConsoleLabel name="overview" /> page,
+click <ConsoleLabel name="service settings" />, go to **Advanced configuration**, and
+click **Configure**.
+
+## Download data
+
+<Tabs groupId="method">
+<TabItem value="console" label="Aiven Console">
+
+1. Log in to the [Aiven Console](https://console.aiven.io/), select your project, and
+   choose your Aiven for InfluxDB service.
+1. On the service's <ConsoleLabel name="overview"/> page, click
+  <ConsoleLabel name="service settings"/>.
+1. In the  **Service management** section, click <ConsoleLabel name="actions"/> >
+   **Download data**.
+1. In the **Download InfluxDB data** window, review the details and enter a name
+   in the **Backup name** field.
+1. Click **Request backup** to create a backup of your latest data.
+
+   :::note
+   The backup process may take several minutes, depending on the size of your data.
```

Comment:
We probably should be more accurate about the backup process duration. For most services (~90%) this phrase is correct, but for the biggest instances it might take up to a few hours.

Reply:
rephrased this.